openedx/openedx-learning
# feat: get_last_publish log for PublishableEntity [FC-0059]

### Description

Adds support for fetching the most recent PublishLog for a given Component, by querying the relevant PublishLogRecords.

### Testing instructions

See https://github.com/openedx/edx-platform/pull/35195 for full testing instructions.
